tests/users/test_api_list_users.py

```diff
@@ -91,7 +91,7 @@ def test_list_users_api_without_search_query_resource_auth(self):
     def test_list_users_api_with_a_search_query_resource_auth(self):
         auth_header = get_test_request_header(self.admin_user.id)
         expected_response = [marshal(self.other_user, public_user_api_model)]
-        actual_response = self.client.get('/users?search=b', follow_redirects=True,
+        actual_response = self.client.get(f'/users?search={self.other_user.name[-3:]}', follow_redirects=True,
```

Comment:
I am actually doubtful in this case, how should we ensure that a substring is unique in this case, in case we consider more test data will be added in future.